src/css/components/_c-404.scss

```diff
@@ -28,7 +28,7 @@
 	}
 
 	@include mappy-bp(lap-small) {
-		padding: 1rem 4rem 2rem 4rem;
```

Comment:
Same here for `padding-left`.

Reply:
same as [prior response](https://github.com/a11yproject/a11yproject.com/pull/984/files/a442075085a0173abbc13b41848ff2affdf3df4c#r464021536) to other review.

Can you advise which direction you prefer (removing the stylelint rule to allow redundant values or leaving this change as is)?